changelog.md

```diff
@@ -1,6 +1,34 @@
 # Changelog for [`unix` package](http://hackage.haskell.org/package/unix)
 
 ## 2.8.0.0 *UNRELEASED*
+  * Fix `semTrywait` on darwin
+
+  * Fix vanishing environment variables in `System.Posix.Env.ByteString.putEnv`/`System.Posix.Env.ByteString.getEnv`
+
+  * Add missing `setEnvironment` and `clearEnv` to `System.Posix.Env.ByteString`
+
+  * Add support for filepath-1.4.100.0 with `PosixString` variants
+
+  * Expose `DirStream`, `CDir`, `CDirent` and `DirStreamOffset` (internal API)
+
+  * Add `fdRead`/`fdWrite` with ByteString payload and deprecate String based `fdRead`
+
+  * Add `clocksPerSec`
+
+  * support wasm32-wasi
+
+  * Implement BaudRate as an `Int` newtype and allow matching via extensible PatternSynonyms
```

Comment:
More precisely, it is a newtype for `CSpeed` from https://hackage.haskell.org/package/base-4.16.2.0/docs/System-Posix-Types.html#t:CSpeed